### PR TITLE
Add arithmetic operations, bound operations, make names uniform

### PR DESCRIPTION
**Pull request by miniBill**

### CODE REVIEW

File: src/Bound.elm [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+            value b
+    in
+    if x < y then
+        a
+
+    else if y < x then
+        b
+
+    else
+        andInclusives a b
```
Comment (r-k-b) [inline src/Bound.elm:131]: yeah, that's much nicer 😄

File: src/Interval.elm [hunk 2]
```diff
@@ -827,13 +706,66 @@ subtract a b =
...
 
 
-{-| Hold the bound value steady, but invert the open/closed property.
--}
-invertBound : Bound -> Bound
-invertBound b =
-    case b of
-        Inclusive n ->
-            Exclusive n
+plus : Interval -> Interval -> Interval
```
Comment (r-k-b) [inline src/Interval.elm:818]: neat! I hadn't gotten into interval arithmetic before

**Review by r-k-b — APPROVED** [review]

**Comment by miniBill** [comment]
Happy you like the changes 😊

I'll probably open a new PR in the future to add division and multiplication

**Comment by miniBill** [comment]
Looking forward to the release :eyes:

**Pull request merged**